src/main/resources/application.yaml

```diff
@@ -89,6 +89,9 @@ notifications:
     claimantResponseNo: ${TSE_CLAIMANT_RESPONSE_NO:0727a137-322f-458b-9941-ee8b6d3c11c2}
     tribunalResponse: ${TSE_TRIBUNAL_RESPONSE:9da290ff-c7da-4897-80d2-114361271dfa}
     respondentResponse: ${TSE_RESPONDENT_RESPONSE:63b0488f-134c-4bf2-8dea-40df4d158628}
+    tribunalResponseToRequest: ${TSE_TRIBUNAL_RESPONSE_TO_REQUEST:a90587ee-61a4-430a-b14a-5801c14b4976}
```

Comment:
Since it's yaml you can take out the common part into a new title.

Reply:
It's quite confusing for example why there's RequestYes, RequestNo and Request by itself

Reply:
I see it matches convention that already exists, don't worry